.github/actions/test-blueprint/action.yml

```diff
@@ -0,0 +1,54 @@
+# This is a test blueprint action that will be called from
+# test.yml, where the actual CI is invoked from.
+# It will call the tests defined in the flake.nix file.
+# These are: common-tests, gc-tests, and tests (debug, release)
+# They all take the same install steps, use cachix, and a nix
+# command like the following:
+    # - run: nix profile install .#nix-build-uncached
+    #   - name: "Run common tests"
+    #     run: |
```

Comment:
is this a `run` in a `run`? Somewhat confusing the hell out of me...

Reply:
looks like a wrong indent, it should probably be two separate commands

Reply:
ah, yes, it's a wrong indent in the copy-paste comment -- I'll fix